express_import/main.ipynb

```diff
@@ -0,0 +1,185 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "55dc7fa8-aa4c-4748-8a2f-c98b030e22fe",
+   "metadata": {},
+   "source": [
+    "## Express Import API Call\n",
+    "This notebook illustrates an example to directly import an S3 asset and add it into a task in a project in one call. This example uses:\n",
+    "\n",
+    "- `requests` library for API calls"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "257a0f4c-ccc4-46d8-acbe-5c6e51e4aad8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import required libraries\n",
+    "import requests"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6a220865-60c8-4b47-afb1-b2a21185ebf3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# API URLs\n",
+    "auth_url = \"https://api.centaurlabs.com/auth/public/v1/login\"\n",
+    "tasks_url = \"https://api.centaurlabs.com/tasks/public/v1/list?project_id=\"\n",
```

Comment:
The project_id query parameter can be added directly in the request instead of as part of the URL:
```suggestion
    "tasks_url = \"https://api.centaurlabs.com/tasks/public/v1/list\"\n",
```